espflash/src/image_format/mod.rs

```diff
@@ -26,12 +31,70 @@ const WP_PIN_DISABLED: u8 = 0xEE;
 
 #[derive(Debug, Clone, Copy, Pod, Zeroable)]
 #[repr(C, packed)]
-struct EspCommonHeader {
+#[doc(alias = "esp_image_header_t")]
```

Comment:
Maybe we could point to https://docs.espressif.com/projects/esptool/en/latest/esp32c3/advanced-topics/firmware-image-format.html#file-header and https://docs.espressif.com/projects/esptool/en/latest/esp32c3/advanced-topics/firmware-image-format.html#extended-file-header?

Reply:
Done